tardis/montecarlo/tests/test_packet_source.py

```diff
@@ -19,8 +19,7 @@ def packet_unit_test_fpath(tardis_ref_path):
 
 
 def test_bb_packet_sampling(request, tardis_ref_data, packet_unit_test_fpath):
-    bb = BlackBodySimpleSource(2508)
-    rng = np.random.default_rng(seed=1963)
+    bb = BlackBodySimpleSource(1963)
```

Comment:
This is the only place in my changes where the changes might affect legacy test cases due to a different seed.